test/run-verify-host.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```

Comment:
`test/run-verify-host-user.sh` and ` test/run-verify-host.sh` names are a bit confusing. I expected one to run as `user` and the other as `root` doing the same or something like that. Which is kind of true, but still confusing at first to discover what does one script and what does the other one.

Reply:
I'll gladly rename them to something more sensible. Any ideas? Maybe verify-host-{setup,run} ?

There's more work to be done for F34 (see https://github.com/martinpitt/cockpit/pull/13), and since with FMF the script name is not visible/relevant from outside any more, we can rename this at will.